src/lint/linter/WebCliLinter.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * Uses UberStandard to detect errors and potential problems in JavaScript code.
+ */
+final class WebCliLinter extends ArcanistExternalLinter {
+
+  private $eslintrc;
+
+  public function getInfoName() {
+    return 'WebCli';
+  }
+
+  public function getInfoURI() {
+    return 'https://github.com/uber/standard';
```

Comment:
I think it is fine then. Just wanted to be sure it is not leftover from some copy&pasted code.